src/test/java/net/pms/formats/FormatsRegressionTest0.java

```diff
@@ -11,86 +11,49 @@
 @SuppressFBWarnings({ "DLS_DEAD_LOCAL_STORE", "MS_SHOULD_BE_FINAL" })
 public class FormatsRegressionTest0 {
 
-	public static boolean debug = false;
-
 	@Test
 	public void test01() throws Throwable {
-
-		if (debug) {
-			System.out.format("%n%s%n", "FormatsRegressionTest0.test01");
-		}
-
 		int i0 = net.pms.formats.Format.UNKNOWN;
 
 		// Regression assertion (captures the current behavior of the code)
 		org.junit.Assert.assertTrue(i0 == 8);
-
 	}
 
 	@Test
 	public void test02() throws Throwable {
-
-		if (debug) {
-			System.out.format("%n%s%n", "FormatsRegressionTest0.test02");
-		}
-
 		net.pms.formats.audio.ADPCM aDPCM0 = new net.pms.formats.audio.ADPCM();
 		net.pms.formats.Format format1 = aDPCM0.getSecondaryFormat();
 		aDPCM0.setIcon("");
 
 		// Regression assertion (captures the current behavior of the code)
 		org.junit.Assert.assertNull(format1);
-
 	}
 
 	@Test
 	public void test03() throws Throwable {
-
-		if (debug) {
-			System.out.format("%n%s%n", "FormatsRegressionTest0.test03");
-		}
-
 		net.pms.formats.Format format1 = net.pms.formats.FormatFactory.getAssociatedExtension("hi!");
 
 		// Regression assertion (captures the current behavior of the code)
 		org.junit.Assert.assertNull(format1);
-
 	}
 
 	@Test
 	public void test05() throws Throwable {
-
-		if (debug) {
-			System.out.format("%n%s%n", "FormatsRegressionTest0.test05");
-		}
-
 		net.pms.formats.Format format1 = net.pms.formats.FormatFactory.getAssociatedFormat("");
 
 		// Regression assertion (captures the current behavior of the code)
 		org.junit.Assert.assertNull(format1);
-
 	}
 
 	@Test
 	public void test06() throws Throwable {
-
-		if (debug) {
-			System.out.format("%n%s%n", "FormatsRegressionTest0.test06");
-		}
-
 		net.pms.formats.MKV mKV0 = new net.pms.formats.MKV();
 		net.pms.formats.audio.RA rA1 = new net.pms.formats.audio.RA();
-		mKV0.setSecondaryFormat((net.pms.formats.Format) rA1);
-
+		mKV0.setSecondaryFormat(rA1);
```

Comment:
If you don't mind explain me, at least tried ;)
What that is supposed do ?

**Note**: Don't worry, i'll not ask every time, but if i understand only this test it's already good :P